students/alexander_boone/lesson04/assignments/mailroom.py

```diff
@@ -1,24 +1,25 @@
 #!/usr/bin/env python
 import os
-
-# global variables
+import sys
 
 # prepare initial data structure for donor data
-donors = [{'first_name': 'Arnold', 'last_name': 'Schwarzenegger', 'donation_total': 100000, 'donation_count': 1}, {'first_name': 'Lebron', 'last_name': 'James', 'donation_total': 1000000, 'donation_count': 1}, {'first_name': 'Elon', 'last_name': 'Musk', 'donation_total': 2000000, 'donation_count': 1}, {'first_name': 'Walter', 'last_name': 'White', 'donation_total': 500000, 'donation_count': 1}, {'first_name': 'Gordon', 'last_name': 'Ramsay', 'donation_total': 1280000, 'donation_count': 1}]
-# donations = [100000, 1000000, 2000000, 500000, 1280000]
-# donation_count = [1, 1, 1, 1, 1]
-donor_data = list()
-for i in range(len(donors)):
-    #donor_data.append((donors[i], [donations[i], donation_count[i]]))
-    donor_data.append(donors[i])
-
-# Note: donor_data is a LIST of TUPLES with each tuple containing (dict of first/last name, [donation total, donation count])
+donors = [{'first_name': 'Arnold', 'last_name': 'Schwarzenegger',
+           'donation_total': 100000, 'donation_count': 1},
+          {'first_name': 'Lebron', 'last_name': 'James',
+           'donation_total': 1000000, 'donation_count': 1},
+          {'first_name': 'Elon', 'last_name': 'Musk',
+           'donation_total': 2000000, 'donation_count': 1},
+          {'first_name': 'Walter', 'last_name': 'White',
+           'donation_total': 500000, 'donation_count': 1},
+          {'first_name': 'Gordon', 'last_name': 'Ramsay',
+           'donation_total': 1280000, 'donation_count': 1}]
```

Comment:
this is still using a list as a data structure, not a dictionary.


as per my previous review, it would look like this:
```
donors = {"Bob": [100]}
```

do you have further questions on what it would look like as a dictionary?

Reply:
I believe I understand now - I'm now using:

`{"Bob Smith" : [100, 2] , "Monty Python" : [400, 5]}`

as my format, where [100, 2] is a list for each donor containing [total donations, donation count]. How does this sound?

Reply:
One more question:

I'm using pycodestyle as my checker for PEP 8. Will this ensure that I'm meeting all guidelines, or does pycodestyle miss some formatting that is required?

Reply:
@ alexanderboone96 yes on your question about donor data structure and I am not personally familiar with `pycodestyle` but I believe it should.